tests/test_parser.py

```diff
@@ -1207,3 +1207,18 @@ def test_parse_identifier_or_str_literal(input: str, expected: str | None):
             parser.parse_identifier_or_str_literal()
     else:
         assert parser.parse_identifier_or_str_literal() == expected
+
+
+@pytest.mark.parametrize(
+    "input, expected",
+    [
+        ("2x3x4", [2, 3, 4]),
```

Comment:
Can you please add the empty case here?
```suggestion
    [
        ("", []),
```

Reply:
Oh and probably `x`